### ISSUE TITLE
fairseq/examples/roberta/README.pretraining.md

### ISSUE THREAD

**Issue by yucoian** [pytorch/fairseq#1073]
When I followed this tutorial for pretraining the Roberta on a 16GPUs machine, I found it didn't output a step result, like a loss, ppl etc.  However, the GPUs and CPUs utilization both are 100%.  Is there any solutions?

**Comment by lematt1991** [comment]
Closing due to inactivity.  Please open a new issue if you are still having problems.
